Tools/TestWebKitAPI/Tests/WGSL/ParserTests.cpp

```diff
@@ -91,6 +88,21 @@ TEST(WGSLParserTests, Struct)
     EXPECT_EQ(memberType.name(), "i32"_s);
 }
 
+TEST(WGSLParserTests, Struct)
+{
+    // without trailing comma
+    testStruct(
+        "struct B {\n"
+        "    a: i32\n"
+        "}"_s);
+
+    // with trailing comma
+    testStruct(
+        "struct B {\n"
+        "    a: i32,\n"
```

Comment:
Is it just the last one for which the trailing comma is optional?

Reply:
Yes, only the last one